components/match2/wikis/trackmania/brkts_wiki_specific.lua

```diff
@@ -0,0 +1,22 @@
+---
+-- @Liquipedia
+-- wiki=trackmania
+-- page=Module:Brkts/WikiSpecific
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lua = require('Module:Lua')
+local Table = require('Module:Table')
+
+local WikiSpecific = Table.copy(Lua.import('Module:Brkts/WikiSpecific/Base', {requireDevIfEnabled = true}))
+
+function WikiSpecific.getMatchGroupContainer(matchGroupType)
+	return matchGroupType == 'matchlist'
+		and Lua.import('Module:MatchGroup/Display/Matchlist', {requireDevIfEnabled = true}).MatchlistContainer
+		or Lua.import('Module:MatchGroup/Display/Bracket/Custom', {requireDevIfEnabled = true}).BracketContainer
+end
```

Comment:
```suggestion
```

Reply:
did not understand why you want to remove this entire function. If you remove it then we can just remove BrktsSpecific nah ? Cause it's just a copy of the Base one

Reply:
if we kick `Module:MatchGroup/Display/Bracket/Custom` (sicne it is not needed) then we have to kick this function too else we will get errors

and you can not juist use [`Module:Brkts/WikiSpecific` from commons](https://liquipedia.net/commons/Module:Brkts/WikiSpecific) since that (for i don't remember which reason) overwrites the processMatch function and hence kills the call of `Module:MatchGroup/Input/Custom`